.github/workflows/ci_web.yml

```
@@ -29,16 +29,13 @@ jobs:
       - name: Pack
         if: github.ref == 'refs/heads/main' || github.ref == 'refs/heads/release'
         run: mv dist reearth-cms-web && tar -zcvf reearth-cms-web.tar.gz reearth-cms-web
 
-      # TODO: Remove after dockerizing the web.
       - uses: actions/upload-artifact@v4
         if: github.ref == 'refs/heads/main' || github.ref == 'refs/heads/release'
         with:
           name: reearth-cms-web
           path: web/reearth-cms-web.tar.gz
-          if-no-files-found: error
+          if-no-files-found: error
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)